core/plugins/org.polarsys.capella.core.sirius.analysis/src/org/polarsys/capella/core/sirius/analysis/CsServices.java

```diff
@@ -6949,4 +6949,13 @@ else if (target instanceof FunctionalChainInvolvementFunction) {
   public boolean isActor(EObject context) {
     return ComponentExt.isActor(context);
   }
+  
```

Comment:
Can rename this function to smth like: getConstraintLocation.

Reply:
Done